modules/upgrade/pages/db-migration.adoc

```diff
@@ -4,10 +4,10 @@
 This is about PostgreSQL database migration.
 
 If you want to upgrade {productname} 3.2 to {productname} 4 you must check whether {productname} 3.2 is already based on PostgreSQL version 10.
```

Comment:
Not for this PR: I'd rewrite this doc so it fits both SUSE Manager or Uyuni as:

```
If you want to upgrade to the latest {productname} version you must check whether {productname} Server is already based on PostgreSQL version 10.
```

And then below, remove all mentions to `3.2` except where it's REALLY needed (and in that case enclose that with a diff).